Feat: Review Spelling of Listing Code

danOIntellitect opened the pull request:
## Summary

Added a spell check editor config along with a exlusion.dic file that contains the strings that we determine are spelled correctly. Then I went through the source code and corrected the misspelled words.

Fixes 1618 from the other repo : https://github.com/IntelliTect-dev/EssentialCSharp.Tooling/issues/1618

danOIntellitect committed:
feat: Reviewing spelling of listing code

In the process of checking spelling for the listing code. Trying to find a better way to automate this process such as making spelling errors actual warnings

danOIntellitect committed:
feat: Finished spell checking the chapters

danOIntellitect committed:
fix: Remove the conflicting listings as these have been deleted

Keboo commented on .editorconfig:
```diff
@@ -0,0 +1,9 @@
+# Remove the line below if you want to inherit .editorconfig settings from higher directories
```
nit, I would start with the default editorconfig. You can generate one by running `dotnet new editorconfig` from the root of the repo.

Keboo commented on exclusion.dic:
```diff
@@ -0,0 +1,68 @@
+vmax
+decryptor
+Abidal
+Autoproperty
```
Can we drop this and fix the usage to be "AutoProperty"?

Keboo commented on exclusion.dic:
```diff
@@ -0,0 +1,68 @@
+spokane
+urls
+iassist
+worldvision
```
Is it possible to drop this and fix usage to be `worldVision`?

danOIntellitect replied:
<img width="180" alt="image" src="https://github.com/IntelliTect/EssentialCSharp/assets/139163393/3fbe37ac-48d3-49e6-aae8-88a859142929">
Here is the usage, I would argue to keep it worldvision since it is a link. But capital V still works for the link

danOIntellitect replied:
I made the change in the most recent commit, but did it separately if we want to roll it back

Keboo commented on exclusion.dic:
```diff
@@ -0,0 +1,68 @@
+inigo
+Machava
+Uncompress
+firstname
+lastname
```
Can we drop these in favor of change usage to have a capital N for name?

Keboo commented on src/Chapter01/Listing01.22.SampleCILOutput.cs:
```diff
@@ -58,7 +58,7 @@ .maxstack 8
 
 
-  .method public hidebysig specialname rtspecialname instance void .ctor() cil managed
+  .method public hidebysig specialName rtspecialname instance void .ctor() cil managed
```
Sorry I was wrong when I told you to make this change, I did not realize it was an CIL identifier. This should be all lowercase.

danOIntellitect committed:
fix: Pr review feedback

Made the editor config contain all default dotnet values, changed some different words for the casing

danOIntellitect committed:
fix: 6.55 was not meant to be changed

danOIntellitect committed:
Merge branch 'v12.0' into DO_1618_SpellChecking

danOIntellitect committed:
fix: Change worldvision link
